features/guidance/new-name.feature

```diff
@@ -125,10 +125,10 @@ Feature: New-Name Instructions
             |   |   |   |   | c |
 
         And the ways
-            | nodes  | highway     |
-            | ab     | residential |
-            | bc     | residential |
-            | bd     | service     |
+            | nodes  | highway     | oneway |
```

Comment:
simply makes it easier to debug and does not change the tested instructions.
If we have oneways, only one intersection is considered in the output. Otherwise we get three.